### PR TITLE
Update image, text, and link for the new job promotion.

### PR DESCRIPTION
**Pull request by AliceStatler**

### CODE REVIEW

File: _data/instagram.yml [hunk 1]
```diff
@@ -4,6 +4,6 @@ blog:
   link: /blog/tech-stories/5-most-valuable-things-i-learned-in-my-10-plus-year-career/
 
 job:
-  image: /assets/img/pages/headers/public/journey-developer.jpg
-  text: We're hiring a<br /><strong>Ruby Developer (m/f/d)</strong>
-  link: https://epages-gmbh.jobs.personio.de/job/202168?language=en
+  image: /assets/img/pages/headers/public/onboarding.jpg
+  text: We're hiring a<br /><strong style="font-size:2.0rem;">Software Developer - 3rd Party Integration, REST API (m/f/d)</strong>
```
Comment (ywolpers) [inline _data/instagram.yml:8]: Did you get the "style" part from Team 42 or have it reviewed by them?

Comment (AliceStatler) [reply _data/instagram.yml:8]: No, but I've tested it myself. Due to the long title, this font size looks best.

Comment (AliceStatler) [reply _data/instagram.yml:8]: If you want to I can show you the result tomorrow.

Comment (ywolpers) [reply _data/instagram.yml:8]: If you've tested it and you're sure that it works, that is fine for me :)

**Pull request merged**